content/docs/gke/troubleshooting-gke.md

```diff
@@ -417,3 +417,53 @@ kubectl -n kubeflow create secret generic kubeflow-oauth \
        --from-literal=client_id=${CLIENT_ID} \
        --from-literal=client_secret=${CLIENT_SECRET}
 ```
+
+## Managed certificates can fail if service management API not enabled
+
+This section describes how to enable service management API to avoid managed certificates failure.
+
+To check your certificate: 
+1. Run the following command:
+
+```
+kubectl -n istio-system describe managedcertificate gke-certificate
+```
+
+Make sure the certificate status is either `Active` or `Provisioning` which means it is not ready. For more details on certificate status, refer to the [certificate statuses descriptions](https://cloud.google.com/load-balancing/docs/ssl-certificates?hl=en_US&_ga=2.164380342.-821786221.1568995229#certificate-resource-status) section. Also, make sure the domain name is correct.
+
+2. Run the following command using the certificate name from the previous step:
```

Comment:
We prefer to use `1.` for all list items, rather than `1.`, `2.`, and so on. Using `1.` works just as well, and it's more flexible when we want to add/remove list items in the future.

Reply:
LGTM.
I fixed it to conform to other Kubeflow pages formatting rules.